terraform/groups/ecs-service/main.tf

```diff
@@ -54,18 +55,37 @@ module "ecs-service" {
   # Service configuration
   service_name = local.service_name
   name_prefix  = local.name_prefix
-  use_fargate  = var.use_fargate
 
   # Service Healthcheck configuration
 
   # Service performance and scaling configs
-  desired_task_count = var.desired_task_count
-  required_cpus      = var.required_cpus
-  required_memory    = var.required_memory
+  desired_task_count                 = var.desired_task_count
+  max_task_count                     = var.max_task_count
+  required_cpus                      = var.required_cpus
+  required_memory                    = var.required_memory
+  service_autoscale_enabled          = var.service_autoscale_enabled
+  service_autoscale_target_value_cpu = var.service_autoscale_target_value_cpu
+  service_scaledown_schedule         = var.service_scaledown_schedule
+  service_scaleup_schedule           = var.service_scaleup_schedule
+  use_capacity_provider              = var.use_capacity_provider
+  use_fargate                        = var.use_fargate
+  fargate_subnets                    = local.application_subnet_ids
+
+  # Cloudwatch
+  cloudwatch_alarms_enabled = var.cloudwatch_alarms_enabled
 
   # Service environment variable and secret configs
-  task_environment = local.task_environment
-  task_secrets     = local.task_secrets
+  task_environment          = local.task_environment
+  task_secrets              = local.task_secrets
+  app_environment_filename  = local.app_environment_filename
+  use_set_environment_files = local.use_set_environment_files
 
-  depends_on=[module.secrets]
-}
+  # eric options for eric running API module
+  use_eric_reverse_proxy    = true
```

Comment:
From my understanding, the web applications do not need the ERIC reverse proxy. I thought only the backend APIs need the eric proxy container into it.